.github/workflows/rmc.yml

```diff
@@ -42,17 +41,24 @@ jobs:
             --set=rust.deny-warnings=false
 
       - name: Update submodules
-        run: git submodule update --init
-
-      - name: Export backtrace flags
-        run: export RUST_BACKTRACE=1
+        run: |
+          # Since we download cached artifacts, we can skip checking out llvm locally in CI
+          git config submodule."src/llvm-project".update none
+          git submodule update --init --depth 1
 
       - name: Build RMC
-        run: ./x.py build -i --stage 1 library/std
+        run: |
+          export RUST_BACKTRACE=1
```

Comment:
you could also use env to set the environment variable: https://docs.github.com/en/actions/learn-github-actions/environment-variables

It can be done in a workflow, job or step level.